django_dramatiq/apps.py

```diff
@@ -24,6 +24,7 @@
         "django_dramatiq.middleware.DbConnectionsMiddleware",
     ]
 }
+DEFAULT_TASK_ENCODER = "dramatiq.encoder.JSONEncoder"
```

Comment:
Thanks! Please rename this and the setting name to `DEFAULT_ENCODER` and I'll get this merged.

Reply:
:+1: